rpcs3/Emu/RSX/Overlays/overlay_list_view.cpp

```diff
@@ -56,7 +56,19 @@ namespace rsx
 
 		void list_view::update_selection()
 		{
-			auto current_element = m_items[m_selected_entry * 2].get();
+			if (m_selected_entry < 0)
+			{
+				return;
+			}
+
+			const size_t current_index = static_cast<size_t>(m_selected_entry) * 2;
+
+			if (m_items.size() <= current_index)
```

Comment:
Is this really ok? Shouldn't we change the selected entry in this case?

Reply:
The actual bug was already fixed by adding the `else` below.
So this is purely sanity checking at this point.

Reply:
Imagine someone calling this with an empty list and we start resetting m_selected_entry to 0 or -1....
This would end up being an infinite loop.

Reply:
or did you mean setting it to -1 ?

Reply:
set to last item or none if list is empty. Or does it already happen automatically? Its strange to imagine deleting the last item from the list and then losing the selection cursor.

Reply:
as I said... it's just a sanity check.
might as well throw an exception there